### DISCUSSION TITLE
[Feature Request] Vertical tabs

### DISCUSSION THREAD

**Discussion by rayaanr** [heroui-inc/heroui#1568] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently only horizontal tabs are available. We like to use vertical tabs also

### Describe the solution you'd like

It is good to have vertical tabs [like this](https://mui.com/material-ui/react-tabs/#vertical-tabs)

### Screenshots or Videos

<img width="733" alt="image" src="https://github.com/nextui-org/nextui/assets/108685206/68a06350-d2c9-476d-80e6-c133af94c3bd">

**Comment by abbystarnes** [comment]
@jrgarciadev Hey! I saw you converted this from an issue to a discussion. Is this feature request gaining any traction? Would you accept a PR to modify your tabs component to accommodate vertical tabs? TIA!

**Comment by Fontalvo22** [comment]
You can make it with this css rules on our css:
`.scrollbar-hide {
    display: flex;
    flex-direction: column;
}`

**Comment by DanBoskovic** [comment]
Listbox is sort of that already, but needs a bit of customisation to work.

**Comment by Tarkan-Zarrouk** [comment]
Looks like this has been available for a while [VerticalTabs](https://nextui.org/docs/components/tabs#vertical)
